### PR TITLE
17 firebase setup for the application

### PR DESCRIPTION
**Pull request by tanishq-2208**

### CODE REVIEW

File: pubspec.yaml [hunk 1]
```diff
@@ -38,6 +38,15 @@ dependencies:
   # Use with the CupertinoIcons class for iOS style icons.
   cupertino_icons: ^1.0.6
   iconify_flutter: ^0.0.5
+  firebase_core: ^2.31.0
+  flutter_bloc: ^8.1.1
+  hydrated_bloc: ^9.0.0
+  intl: ^0.19.0
+  path_provider: ^2.0.9
+  equatable: ^2.0.3
```
Comment (maheshalyana) [inline pubspec.yaml:46]: Are we using these packages

**Review by maheshalyana — CHANGES_REQUESTED** [review]

File: pubspec.yaml (hunk 1 above)
Comment (tanishq-2208) [reply pubspec.yaml:46]: Its better to use them....these help in enhancing the functionality of our flutter application.

Comment (maheshalyana) [reply pubspec.yaml:46]: Yeah I know, but I don't think we are going to use them in app now, when we'll be using then we can add them if needed

Comment (tanishq-2208) [reply pubspec.yaml:46]: okay

**Review by maheshalyana — APPROVED** [review]

**Pull request merged**